evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -145,6 +146,26 @@ def test_set_forecast(get_ts_X_y):
     assert fh_.is_relative
 
 
+def test_get_sp():
```

Comment:
again for my clarity - does the last case here cover the `None` case? Might be helpful to add a comment here.